site/content/docs/developing/overrides/yamlpatch.en.md

```diff
@@ -4,7 +4,7 @@
 
 ## When should I use YAML Patch over CDK overrides?
 
-Both options are a "break the glass" mechanism to access and configure functionality that is not surfaced by Copilot [manifests](../../manifest/overview.en.md).
+Both options are a "break the glass" mechanism to access and configure functionality that is not surfaced by Copilot [manifests](../../manifest/overview.md).
```

Comment:
Wait, why did these two `en`s come out?

Reply:
oh! lol. i should have said why 😛. apparently you don't need the language extension in the links!!! So, for instance - there isn't a `pipeline-override.ja.md` right now, so if a user was on the `ja` version of the site, they would see that page in english. But, since the link to `yamlpatch.md` is generic, the link will take them to `yamlpatch.ja.md` instead of forcing them to the `en` version.